cmake/configs/nuttx_px4fmu-v4_default.cmake

```diff
@@ -5,7 +5,8 @@ px4_nuttx_configure(HWCLASS m4 CONFIG nsh ROMFS y ROMFSROOT px4fmu_common)
 set(board_serial_ports
 	GPS1:/dev/ttyS3
 	TEL1:/dev/ttyS1
-	TEL2:/dev/ttyS2)
+	TEL2:/dev/ttyS2
+	TEL3:/dev/ttyS0)
```

Comment:
Need to find the appropriate name for this one.
TODO: check pixracer for label

Reply:
in `rcS`
# UART mapping on FMUv2/3/4:
#
# UART1         /dev/ttyS0      IO debug (except v4, there ttyS0 is the wifi,

Reply:
Should I name it IODebug?